test/python/WMCore_t/MicroService_t/MSUnmerged_t/MSUnmerged_t.py

```diff
@@ -0,0 +1,181 @@
+"""
+Unit tests for MicorService/MSUnmerged/MSUnmerged.py module
+
+"""
+from __future__ import division, print_function
+
+import json
+# system modules
+import os
+import unittest
+
+# WMCore modules
+from WMCore.MicroService.MSUnmerged.MSUnmerged import MSUnmerged, MSUnmergedRSE
+from WMCore.Services.Rucio import Rucio
+
+
+def getTestFile(partialPath):
+    """
+    Returns the absolute path for the test json file
+    """
+    normPath = os.path.normpath(os.path.join(os.path.dirname(__file__), '..', '..', '..', '..'))
+    return os.path.join(normPath, partialPath)
+
+
+class RucioConMonEmul(object):
```

Comment:
@todor-ivanov Todor, once things are running smoothly in production, I would suggest to convert this class to a real RucioConMon emulator under: https://github.com/dmwm/WMCore/tree/master/src/python/WMQuality/Emulators

Could you please make a GH issue for this such that we do not lose track of it? Thanks